src/App.css

```diff
@@ -1,3 +1,65 @@
+.container {
+  color: #271973;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+h1 {
+  font-weight: bolder;
+}
+
 .form-select {
   width: auto;
+}
+
+.movesCounter {
```

Comment:
Whether you use `camelCase` or `kebab-cake` or `snake_case` for CSS class names, I think the most important think is to be consistent. I think it's most common to see `kebab-case`, so that's never a bad choice.